src/ucp/rndv/rndv_put.c

```diff
@@ -184,8 +190,8 @@ ucp_proto_rndv_put_common_request_init(ucp_request_t *req)
 {
     const ucp_proto_rndv_put_priv_t *rpriv = req->send.proto_config->priv;
 
-    req->send.rndv.put.atp_map   = rpriv->atp_map;
-    req->send.rndv.put.flush_map = rpriv->flush_map;
+    req->send.rndv.put.flush_lane = 0;
+    req->send.rndv.put.atp_lane   = 0;
```

Comment:
IMO it's adding extra overhead that is not really needed, and may be even confusing since flush_lane/atp_lane are not really the exact lanes, but a lower bound on the next lane